Tests/TestCommon.cs

```diff
@@ -17,6 +17,7 @@ static TestCommon()
             // Read configuration data
             TenantUrl = ConfigurationManager.AppSettings["SPOTenantUrl"];
             DevSiteUrl = ConfigurationManager.AppSettings["SPODevSiteUrl"];
+            Dev2SiteUrl = ConfigurationManager.AppSettings["SPODev2SiteUrl"];
```

Comment:
Please do not add another property in app.config. This would require a change by everyone in order to run the unit tests succesfully. See my other comment about using subwebs for testing.

Reply:
Thanks for the comments Erwin. I was a little unsure about these too. The problem I found is that the issue only kicks in when moving between site collections so can't fully test without that. Could it be assumed as the same as the main site collection but with _2 on it?